src/Microsoft.AspNet.Razor.Runtime/TagHelpers/ITagHelper.cs

```diff
@@ -17,6 +17,16 @@ public interface ITagHelper
         int Order { get; }
 
         /// <summary>
+        /// Initializes the <see cref="ITagHelper"/> with the given <paramref name="context"/>.
+        /// </summary>
+        /// <param name="context">Contains information associated with the current HTML tag.</param>
+        /// <remarks>Any additions to <see cref="TagHelperContext.Items"/> should be done within this method to ensure
+        /// they're added prior to executing the children content. When more than one <see cref="ITagHelper"/> runs
```

Comment:
Pick one: `executing the children` or `determining the child content`. "children content" is a mangle.